__mocks__/npm-name.js

```diff
@@ -0,0 +1,19 @@
+// Mocks the module "npm-name"
+
+/**
+ * Resolves true if the package name is not fould on npm.
+ *
+ * @param {String} packageName - The package name to check
+ * @return {Promise} - Resolves false if set as "INVALID"
+ */
+const npmName = packageName => new Promise(resolve => {
+  process.nextTick(() => {
+    if (packageName === 'INVALID') {
+      resolve(false);
+    } else {
+      resolve(true);
+    }
```

Comment:
Nice catch. I'm sure there's plenty of other inconsistencies in my dirty code. 👍